_layouts/default.html

```diff
@@ -9,6 +9,7 @@
   {% include head.html %}
 
   <body>
+    {% include gtm_iframe.html %}
```

Comment:
How important is it that this be right inside the `<body>` tag?

From the perspective of taking this approach and scaling it out as a best practice across many government sites, it would be ideal to be able to keep all the relevant tags and code in one place. I think this is a bigger priority than following Google's recommendations for performance optimization.